src/app/@ansyn/menu-items/cases/components/cases-table/cases-table.component.less

```diff
@@ -108,6 +108,9 @@ div.cases-table {
 				left: 0;
 				color: #ebf8fa;
 			}
+			/deep/ .slider-text {
```

Comment:
why not inside autosave component ?

Reply:
because we wanted the hover style to be inherited by the child component